### ISSUE TITLE
$HOME is not defined

### ISSUE THREAD

**Issue by timclark97** [envkey/envkey#21]
Using "envkey": "^2.0.8" in my typescript app and getting the error `$HOME is not defined` when running it in locally. I am on an m1 mac. When I run `printenv` I see the `HOME` variable is set. I can also run `console.log(process.env.HOME)` from my app and see the correct file path.

package.json script
`"dev": "NODE_ENV=development nodemon --exec ts-node src/index.ts"`

src/index.ts content
```
console.log(process.env.HOME)
import "envkey"
export {}
```

output
```
/Users/myDir
2022/06/20 21:13:15 $HOME is not defined
echo 'error: $HOME is not defined'; false

2022/06/20 21:13:15 $HOME is not defined
```

Any thoughts on what could be causing this?

**Comment by danenania** [comment]
Looking into this...

**Comment by danenania** [comment]
Should be fixed in 2.0.9 of envkey-node
